### DISCUSSION TITLE
Installation of the drogon framework

### DISCUSSION THREAD

**Discussion by 45739847** [microsoft/vcpkg#25309] [Polls]
what do you think about a configuration/installation file that configures everything for the development environment

**Comment by autoantwort** [comment]
What do you exactly mean by that?

**Reply by 45739847** [reply]
after facing problems installing vcpkg, I saw that many of them could exist an automated pre-configured solution, with a .bat file or another executable

**Reply by autoantwort** [reply]
I still don't get what you are proposing :/ 
Are you proposing that it should be possible to provide a configuration file for ports instead of using the default one?

**Comment by AugP** [comment]
Are you thinking of something similar to: https://devblogs.microsoft.com/cppblog/vcpkg-artifacts/?
